Dockerfile

```diff
@@ -73,9 +73,14 @@ ARG TARGETPLATFORM
 # lld has issues building static binaries for ppc so prefer ld for it
 RUN set -e; xx-apk add musl-dev gcc libseccomp-dev libseccomp-static; \
   [ "$(xx-info arch)" != "ppc64le" ] || XX_CC_PREFER_LINKER=ld xx-clang --setup-target-triple
-RUN --mount=from=runc-src,src=/usr/src/runc,target=. --mount=target=/root/.cache,type=cache \
-  CGO_ENABLED=1 xx-go build -mod=vendor -ldflags '-extldflags -static' -tags 'apparmor seccomp netgo cgo static_build osusergo' -o /usr/bin/runc ./ && \
+RUN --mount=from=runc-src,src=/usr/src/runc,target=. \
+  --mount=target=/root/.cache,type=cache <<EOT
+  set -ex
+  CGO_ENABLED=1 xx-go build -mod=vendor -ldflags '-extldflags -static' -tags 'apparmor seccomp netgo cgo static_build osusergo' -o /usr/bin/runc ./
   xx-verify --static /usr/bin/runc
+  file /usr/bin/runc
```

Comment:
Why these extra `file` calls?

Reply:
Removed it, was just for extra debug info.